aikau/src/main/resources/alfresco/documentlibrary/AlfBreadcrumb.js

```diff
@@ -122,20 +113,10 @@ define(["dojo/_base/declare",
        * @instance
        * @param {object} evt The click event
        */
-      onClick: function alfresco_documentlibrary_AlfBreadcrumb(evt) {
-         if (this.targetUrl != null)
-         {
-            // If a URL has been provided then clicking will navigate to that URL (this is typically the 
-            // action for the last breadcrumb in the trail)...
-            this.alfPublish("ALF_NAVIGATE_TO_PAGE", { url: this.targetUrl,
-                                                      type: this.targetUrlType,
-                                                      target: this.targetUrlLocation});
-         }
-         else
+      onClick: function alfresco_documentlibrary_AlfBreadcrumb(/* jshint unused:false */ evt) {
+         if (this.publishTopic)
          {
-            this.alfPublish("ALF_NAVIGATE_TO_PAGE", { url: "path=" + this.path,
-                                                      type: "HASH",
-                                                      target: this.targetUrlLocation});
+            this.alfPublish(this.publishTopic, this.publishPayload, (this.publishGlobal !== undefined && this.publishGlobal === true));
```

Comment:
Could just use `this.alfPublish(this.publishTopic, this.publishPayload, (this.publishGlobal === true));`?